packages/server/src/auth/CrossmintAuthServer.ts

```diff
@@ -135,8 +138,11 @@ export class CrossmintAuthServer extends CrossmintAuth {
         try {
             // It's not necessary to call the logout endpoint, but desirable
             if (request != null) {
-                const { refreshToken } = getAuthCookies(request);
-                await this.logoutFromDefaultRoute(refreshToken);
+                const authCookies = getAuthCookies(request);
+                if (authCookies != null) {
+                    const { refreshToken } = authCookies;
+                    await this.logoutFromDefaultRoute(refreshToken);
```

Comment:
nit: can do this since it's not reused
```suggestion
                    await this.logoutFromDefaultRoute(authCookies.refreshToken);
```